### PR TITLE
OneBranch on rel/v2

### PR DESCRIPTION
**Pull request by bgavrilMS**
Move OneBranch scripts from main to rel/v2

### CODE REVIEW

**Review by trwalke — APPROVED** [review]

**Comment by trwalke** [comment]
Ran a build here and double checked the tasks/packages. everything looks good.
https://dev.azure.com/IdentityDivision/IDDP/_build/results?buildId=1307306&view=results

File: build/template-install-dependencies.yaml [hunk 1]
```diff
@@ -23,6 +23,13 @@ steps:
   inputs:
     version: 8.x  
 
+- task: UseDotNet@2
```
Comment (jmprieur) [inline build/template-install-dependencies.yaml:26]: Do we need .NET 9 in rel/v2?

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @bgavrilMS

File: build/template-install-dependencies.yaml (hunk 1 above)
Comment (jennyf19) [reply build/template-install-dependencies.yaml:26]: no. it's only added in master

**Review by jennyf19 — APPROVED** [review]
thanks @bgavrilMS :shipit:

**Pull request merged**